### ISSUE TITLE
Clarify OTLP/gRPC endpoint configuration option handling

### ISSUE THREAD

**Issue by pellared** [open-telemetry/opentelemetry.io#4400]
I've had users ask why the path is ignored for gRPC. Its not clear to new gRPC users that the path is derived from the proto spec for the service. Do you think its worth elaborating on that here and indicating which parts of the URL are honored for gRPC? Or maybe alternatively there's some gRPC doc we can point to?

_Originally posted by @jack-berg in https://github.com/open-telemetry/opentelemetry-specification/pull/3739#discussion_r1391748293_

I think that we can add a reference to https://github.com/grpc/grpc/blob/master/doc/naming.md. We can also add a notice that gRPC endpoint is NOT an URL therefore it does not understand e.g. the "path" URL component.

**Comment by austinlparker** [comment]
This is a documentation issue @svrnm @cartermp please assign appropriately

**Comment by svrnm** [comment]
Yeah, +1 what @tigrannajaryan suggested to have that mentioned in the SDK Configuration page, we might spread it from there to other pages

**Comment by pellared** [comment]
Unassigning myself as I continuously have other stuff which IMO has bigger priority.
